src/fty_discovery_server.cc

```diff
@@ -687,7 +694,16 @@ void ftydiscovery_create_asset(fty_discovery_server_t* self, zmsg_t** msg_p)
                     link.dest = fty::convert<uint32_t>(DBAssets::name_to_asset_id(iname));
                 }
                 auto conn = tntdb::connectCached(DBConn::url);
-                DBAssetsInsert::insert_into_asset_links(conn, self->default_values_links);
+                // In device centric view for sts, add default link for the 2 inputs if default links have only one entry
+                if (self->device_centric && streq(name, "sts") && self->default_values_links.size() == 1) {
+                    std::vector<link_t> links = self->default_values_links;
+                    links.push_back(self->default_values_links[0]);
+                    DBAssetsInsert::insert_into_asset_links(conn, links);
+                }
+                // default treatment
+                else {
```

Comment:
For readability this comment is oblivious and visually breaks the if branches... could you replace or remove it?

Reply:
Ok I remove the comment